peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -3,6 +3,16 @@
 
 {% block title %}{{ document.title }}{% endblock %}
 
+{% block head-meta %}
+  {{ block.super }}
+  <meta property="og:title" content="{{ document.title }}" />
+  <meta property="og:type" content="book" />
+  <meta property="book:release_date" content="{{ document.date|date:'Y-m-d' }}" />
+  {% if document.author %}
```

Comment:
I see a document as a book. Setting the `og:type` as a `book` also allows us to set the `book:release_date`, which is what we want to achieve. Setting a date.

the property `book:release_date` accepts a date format [ISO-8601](https://www.iso.org/iso-8601-date-and-time-format.html) which accepts the format `Y-m-d`

I also set the author if author field exists(Although, it wasnt a requirement in the ticket)

Reply:
All good :)